target/linux/ar71xx/base-files/lib/upgrade/platform.sh

```diff
@@ -451,6 +452,12 @@ platform_check_image() {
 			magic_ver="0200"
 			;;
 		esac
+		
+		case "$board" in
```

Comment:
case "$board" in
tl-wdr6500-v2 || tl-mr22u-v1)

did not work for me,should that be done differently?